src/main/java/io/kestra/plugin/spark/RSubmit.java

```diff
@@ -23,7 +23,7 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-    title = "Submit a R job to remote cluster"
+    title = "Submit a R job to remote cluster."
```

Comment:
```suggestion
    title = "Submit an R job to a remote cluster."
```

Reply:
Changed it to "Submit a R job to a remote cluster."